frontend/src/constants/API.tsx

```diff
@@ -29,9 +29,15 @@ export const PARCELS = (params: IParcelListParams | null) =>
 export interface IParcelDetailParams {
   id: number;
 }
-export const PARCEL_DETAIL = (params: IParcelDetailParams) => `/parcels/${params.id}`;
+
+export interface IUserDetailParams {
+  id: string;
+}
+export const PARCEL_DETAIL = (params: IParcelDetailParams) => `/my/parcel/${params.id}`;
```

Comment:
I think the route is wrong here, look left (`/parcels/${params.id}`)

Reply:
No need to change this line as earlier PRs (mine included) already change it. This branch just need a rebase. 

Note that if you update this line on your branch you might run into conflicts before being able to merge the PR...